src/healpix_extra.c

```diff
@@ -685,7 +902,21 @@ static void sht_wrapper(int spin,int lmax,int nside,int ntrans,flouble **maps,fc
 #endif //_SPREC
 
   sharp_make_triangular_alm_info(lmax,lmax,1,&alm_info);
-  sharp_make_weighted_healpix_geom_info(nside,1,NULL,&geom_info);
+  if(cs->is_healpix)
+    sharp_make_weighted_healpix_geom_info(cs->n_eq,1,NULL,&geom_info);
+  else {
+    // first nrings is total
+    flouble Delta_theta = fabs(cs->Delta_theta);
+    int tot_sphere_rings = round(M_PI / Delta_theta) + 1;
+    flouble theta1 = cs->theta0 - cs->ny * Delta_theta;
+    int first_ring = round(theta1 / Delta_theta) + 1;
+    sharp_make_cc_geom_info_stripe(tot_sphere_rings,
+				   cs->nx, cs->phi0, // DEGREES OR RADIANS??
```

Comment:
Probably should remove this comment at some point, libsharp operates **in radians**.

Reply:
will remove in my next push